record/record.app.js

```diff
@@ -278,6 +280,12 @@
                     });
                 });
 
+                // chaiseConfig.showWriterEmptyRelatedOnLoad takes precedence over heuristics above for $rootScope.showEmptyRelatedTables when true or false
+                // showWriterEmptyRelatedOnLoad only applies to users with write permissions for current table
+                if ($rootScope.reference.canCreate && typeof chaiseConfig.showWriterEmptyRelatedOnLoad === "boolean") {
```

Comment:
Removed check for `$rootScope.modifyRecord`. This value is based on `chaiseConfig.editRecord`. We don't want that chaise config property affecting how this property behaves. If we want to show all related tables to a writer on load, then having that property set to false shouldn't hide the related tables.